maps/MahoganyMart1F.asm

```diff
@@ -96,28 +96,28 @@ MahoganyMart1FLanceUncoversStaircaseScript:
 	writetext MahoganyMart1FLanceSplitUpText
 	waitbutton
 	closetext
-	applymovement MAHOGANYMART1F_LANCE, MovementData_0x6c412
+	applymovement MAHOGANYMART1F_LANCE, MahoganyMart1FLanceGoDownStairsMovement
 	playsound SFX_EXIT_BUILDING
 	disappear MAHOGANYMART1F_LANCE
 	setscene SCENE_MAHOGANYMART1F_NOTHING
 	waitsfx
 	end
 
-MahogayMart1FGrannyScript:
+MahoganyMart1FGrannyScript:
 	faceplayer
 	opentext
 	pokemart MARTTYPE_STANDARD, MART_MAHOGANY_2
 	closetext
 	end
 
-MovementData_0x6c3f6:
+MahoganyMart1FDragoniteTackleMovement:
```

Comment:
Hyper Beam, not Tackle. :P

Reply:
Hi @Rangi42, thanks for the review/comment.

If I recall correctly, the dragonite does a quick "forward-one-square" and "back-one-square" motion, and the dude it's attacking gets knocked back right after. And there's not really any flashes on the screen or visible energy beams or anything like that. The dialog is regarding a hyper beam, but the movements of the characters all seem pretty tackle-y, or like a body-slam.

(I found a let's play of this part of the game: https://www.youtube.com/watch?v=WYs_oVnrZPs&feature=youtu.be&t=923)

So that's what I had based it on.

That said, if you still prefer it be "HyperBeamMovement" I'll put that.

Best,

\- DeeDeeG

Reply:
It's true that there's no fancy Hyper Beam animation, but Lance explicitly names the attack, so given that "Tackle" is a different attack, not just a generic move-forward-and-back description, it feels inaccurate to me.

Reply:
I was thinking of "a tackle," not necessarily the Pokemon move "Tackle."

Also, it plays `SFX_TACKLE` right before the dragonite attack happens.

Here's the relevant part of the script, right after walking into the Mart during the Lance quest:

```assembly
MahoganyMart1FLanceUncoversStaircaseScript:
        pause 15
        opentext
        writetext MahoganyMart1FLanceDragoniteHyperBeamText
        pause 15
        closetext
        playsound SFX_TACKLE
        applymovement MAHOGANYMART1F_DRAGON, MahoganyMart1FDragoniteTackleMovem>
        applymovement MAHOGANYMART1F_BLACK_BELT, MahoganyMart1FBlackBeltKnocked>
        pause 15
        disappear MAHOGANYMART1F_DRAGON
```

So it basically is treated as a tackle everywhere but Lance's dialogue.

Just making double sure, and I will still change it if you want. Wanted to clarify first.

Reply:
It's no big deal, Tackle is fine too.